mirrord/agent/src/steal/connections/filtered.rs

```diff
@@ -672,6 +680,15 @@ where
         self.blocked_requests
             .insert((client_id, id), request.response_tx);
 
+        for client_id in other_client_ids {
+            tx.send(ConnectionMessageOut::LogMessage {
+                client_id,
+                connection_id: self.connection_id,
+                message: LogMessage::warn("Matching HTTP request stolen by another client".into()),
```

Comment:
```suggestion
                message: LogMessage::warn(format!(
                    "An HTTP request was stolen by another user. URI=({:?}), HEADERS=({:?})",
                    request.request.uri(),
                    request.request.headers(),
                )),
```
I'd add request uri and headers for more context, since that's what we filter on

Reply:
Makes sense, wasn't sure what info would be useful to log